### PR TITLE
Fix eval stats

### PR DESCRIPTION
**Pull request by itkovian**

### CODE REVIEW

File: lib/vsc/utils/nagios.py [hunk 1]
```diff
@@ -447,13 +447,13 @@ def _eval(self, **kwargs):
 
         warn, crit = None, None
         for k, v in sorted(processed_dict.iteritems()):
```
Comment (boegel) [inline lib/vsc/utils/nagios.py]: single-letter variables are evil outside of list comprehensions

File: lib/vsc/utils/nagios.py [hunk 2]
```diff
@@ -447,13 +447,13 @@ def _eval(self, **kwargs):
...
-            if NagiosRange(v['critical']).alert(v['value']):
+            if "critical" in v and NagiosRange(v['critical']).alert(v['value']):
```
Comment (boegel) [inline lib/vsc/utils/nagios.py]: where's the unit test that triggers this bug?

Comment (itkovian) [inline lib/vsc/utils/nagios.py]: coming in a future PR

Comment (boegel) [inline lib/vsc/utils/nagios.py]: good joke

File: lib/vsc/utils/nagios.py (hunk 1 above)
Comment (JensTimmerman) [inline lib/vsc/utils/nagios.py]: this is a list comprehension without the []

**Comment by JensTimmerman** [comment]
needs tests

File: lib/vsc/utils/nagios.py (hunk 1 above)
Comment (boegel) [inline lib/vsc/utils/nagios.py]: no it's not... it's a for loop

**Comment by itkovian** [comment]
@JensTimmerman Please rereview

**Pull request merged**